Data/Sys/GameSettings/GED.ini

```diff
@@ -2,6 +2,7 @@
 
 [Core]
 # Values set here will override the main Dolphin settings.
+CPUThread = 0
```

Comment:
Fake-completion is okay with me.  It shouldn't affect single core, which is what users should be using anyway.